### ISSUE TITLE
Disable row selection in DataColumns/DataColumns2

### ISSUE THREAD

**Issue by RM-STW** [maxim-saplin/data_table_2#188]
First of all, I'm new to flutter in the first place, I'm not sure if this is already documented and I just haven't seen it, but here's my issue:

I am using the "async_paginated_data_table2" example, I want to have an async, paginated, sortable list, however, I need to remove the checkboxes, and later on I'm planning on adding flutter widgets within each row, eg: Switches, BoxShapes, etc, and possibly an onPressed( ) action to each row.

The issue I'm having is I've set "showCheckboxColumn: false," in "AsyncPaginatedDataTable2( )", the checkboxes are now disabled, however, I'm still able to select the rows, Is there a way to disable row selection ?.

Or am I doing a completely wrong approach to it for my needs ?.

**Comment by maxim-saplin** [comment]
Try removing onSelect event handler

**Comment by RM-STW** [comment]
Thanks!
That did seem to work, I removed the "onSelectChanged:( ){ }" from the "data_sources" file and now I cannot click on each row.

However the margin where there would be a label with the amount of rows selected seems to remain present, is this a known bug ?.

I removed the range selector so that shouldn't really be the cause.

Row selection disabled:

![image](https://user-images.githubusercontent.com/130577231/235881021-f46ea501-16b5-4347-9b74-03eadf21b624.png)


Row selection enabled:

![image](https://user-images.githubusercontent.com/130577231/235881616-b8050f1c-fdaf-4c78-b5d1-f7698ab1d94e.png)

Element selected:

![image](https://user-images.githubusercontent.com/130577231/235881679-3e51783d-3765-4155-99ce-8fc2c24c7c04.png)
